cob_description/urdf/cob4_sensorring/sensorring.urdf.xacro

```diff
@@ -4,7 +4,7 @@
   <xacro:include filename="$(find cob_description)/urdf/cob4_sensorring/sensorring.gazebo.xacro" />
   <xacro:include filename="$(find cob_description)/urdf/cob4_sensorring/sensorring.transmission.xacro" />
 
-  <xacro:macro name="sensorring" params="parent name *origin">
+  <xacro:macro name="sensorring" params="parent name *origin active:=false">
```

Comment:
It's the scheme that I started for torso and head, i.e. explicitly **activate** joints...